Sources/TableViewDriver.swift

```diff
@@ -226,7 +198,9 @@ extension TableViewDriver: UITableViewDataSource {
         guard let tableViewModel = self.tableViewModel, let cellViewModel = tableViewModel[indexPath] else {
             fatalError("Table View Model has an invalid configuration: \(String(describing: self.tableViewModel))")
         }
-        return tableView.configuredCell(for: cellViewModel, at: indexPath)
+        let cell = tableView.configuredCell(for: cellViewModel, at: indexPath)
+        cell.accessibilityIdentifier = cellViewModel.accessibilityFormat.accessibilityIdentifierForIndexPath(indexPath)
```

Comment:
This used to only be handled by `refreshViews`. By moving this here, we ensure this is always up-to-date.